spec/NewTwitchApi/Resources/UsersApiSpec.php

```diff
@@ -18,90 +18,102 @@ function let(Client $guzzleClient)
     }
 
     function it_should_get_users_follows_page_by_after(Client $guzzleClient, Response $response)
     {
         $guzzleClient->send(new Request('GET', 'users/follows?after=42', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
-        $this->getUsersFollows("TEST_TOKEN", null, null, null, 42)->shouldBeAnInstanceOf(ResponseInterface::class);
+        $this->getUsersFollows('TEST_TOKEN', null, null, null, 42)->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
     function it_should_get_users_follows_by_everything(Client $guzzleClient, Response $response)
     {
         $guzzleClient->send(new Request('GET', 'users/follows?from_id=12345&to_id=98765&first=42&after=99', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
-        $this->getUsersFollows("TEST_TOKEN", '12345', '98765', 42, 99)->shouldBeAnInstanceOf(ResponseInterface::class);
+        $this->getUsersFollows('TEST_TOKEN', '12345', '98765', 42, 99)->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
     function it_should_get_user_with_access_token(Client $guzzleClient, Response $response)
     {
-        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer access-token']))->willReturn($response);
-        $this->getUsers("access-token")->shouldBeAnInstanceOf(ResponseInterface::class);
+        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->getUsers('TEST_TOKEN')->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
     function it_should_get_user_with_access_token_convenience_method(Client $guzzleClient, Response $response)
     {
-        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer access-token']))->willReturn($response);
-        $this->getUserByAccessToken('access-token')->shouldBeAnInstanceOf(ResponseInterface::class);
+        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->getUserByAccessToken('TEST_TOKEN')->shouldBeAnInstanceOf(ResponseInterface::class);
+    }
+
+    function it_should_create_a_follow(Client $guzzleClient, Response $response)
+    {
+        $guzzleClient->send(new Request('POST', 'users/follows?from_id=123&to_id=321', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->createUserFollow('TEST_TOKEN', '123', '321')->shouldBeAnInstanceOf(ResponseInterface::class);
+    }
+
+    function it_should_create_a_follow_with_notifications(Client $guzzleClient, Response $response)
+    {
+        $guzzleClient->send(new Request('POST', 'users/follows?from_id=123&to_id=321&allow_notifications=1', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->createUserFollow('TEST_TOKEN', '123', '321', 1)->shouldBeAnInstanceOf(ResponseInterface::class);
```

Comment:
I would still pass in `true` here. That's what a developer calling the method in their own code would most likely do, since it is a `bool` argument. However, the Guzzle prophecy object on the previous line should still expect `1` since that is the behavior. Pass in `true` to PHP. `1` gets sent in the URL.